README.md

```diff
@@ -167,6 +167,22 @@ If your editor does not recognise the custom `jest-extended` matchers, add a `gl
 import 'jest-extended';
 ```
 
+_Note: When using `ts-jest >= 25.5.0`_
+
+Since the [breaking changes]() in `25.5.0` you also need to update you `tsconfig.json` to include the new `global.d.ts` file in the `files` property like so:
```

Comment:
```suggestion
Since the [breaking changes]() in `25.5.0` you may also need to update your `tsconfig.json` to include the new `global.d.ts` file in the `files` property like so:
```
Works just fine for me with `jest-extended@0.11.5` and `ts-jest@26.1.4` so I would suggest adding a "may" here.

In fact looking at https://github.com/kulshekhar/ts-jest/pull/1562 it seems that the breaking change only applies if you were using the `include` section in tsconfig.json previously.